app/src/components/domain/RiskImminentEvents/Pdc/index.tsx

```diff
@@ -186,6 +308,34 @@ function Pdc(props: Props) {
         [getFootprint],
     );
 
+    const handlePopupClick = useCallback(
+        (feature: mapboxgl.MapboxGeoJSONFeature, lngLat: mapboxgl.LngLat) => {
+            setClickedPointProperties({
+                feature: feature as unknown as ClickedPoint['feature'],
+                lngLat,
+            });
+            return true;
+        },
+        [setClickedPointProperties],
+    );
+
+    const handlePopupClose = useCallback(
+        () => {
+            setClickedPointProperties(undefined);
+        },
+        [setClickedPointProperties],
+    );
+
+    const handleLayerChange = useCallback((value: boolean, name: LayerType) => {
+        if (name === LAYER_CYCLONE_NODES) {
+            handlePopupClose();
+        }
```

Comment:
Why are we closing popup only for node?

Reply:
To make the consistency.
for instance, when a user click on the cyclone track node then want to hide the node layer.